apcd-cms/src/apps/registrations/views.py

```diff
@@ -16,6 +17,7 @@
 
 
 class SubmissionFormView(View):
+    @never_cache
```

Comment:
If cache is not working (I trust @edmondsgarrett), and [not the main goal of the PR](https://github.com/TACC/Core-CMS-Custom/pull/34#issuecomment-1309482551), then should it be removed? (So as not to introduce a non-working feature that may become a misdirection[^1] or a bug[^2].)

[^1]: Someone could think this solves cache (but it doesn't) and respond accordingly (but incorrectly).
[^2]: This may result in non-intuitively inconsistent cache behavior.

Reply:
That's a good point. I'll pull it out for now and make note of it in the JIRA ticket.